tests/e2e/test_rosacli_idp.go

```diff
@@ -680,4 +680,149 @@ var _ = Describe("Edit IDP",
 				Expect(textData).Should(ContainSubstring("Failed to create IDP for cluster '%s': Expected a valid certificate bundle: open %s: no such file or directory", clusterID, invalidCaFilePath))
 			})
 
+		It("can Create multiple HTPasswd IDPs including creating admin flow by the rosacli command - [id:65160]",
+			labels.Medium, labels.Runtime.Day2,
+			func() {
+				if profile.ClusterConfig.ExternalAuthConfig {
+					Skip("IDP is not supported for external auth")
+				}
+
+				var (
+					idpNames   = []string{"htpasswdn1", "htpasswdn2"}
+					usersValue = "testuser:as65160-MSV5R-bUwmc-2506"
+					idpType    = "htpasswd"
+				)
+
+				By("Test the scenario: create admin --> create htpasswd idp with --users (htpasswdidp1) --> create htpasswd idp with --from-file (htpasswdidp2)")
```

Comment:
Having different scenarios into one case, I would separate in several cases ...

Reply:
Maybe one `Context` and 2 `It` ?

Reply:
done, could you please take a look again? Thanks